pkg/virt-operator/resource/generate/components/scc.go

```diff
@@ -74,7 +74,14 @@ func NewKubeVirtControllerSCC(namespace string) *secv1.SecurityContextConstraint
 	scc.SELinuxContext = secv1.SELinuxContextStrategyOptions{
 		Type: secv1.SELinuxStrategyRunAsAny,
 	}
-	scc.AllowedCapabilities = []corev1.Capability{"SYS_NICE"}
+	scc.AllowedCapabilities = []corev1.Capability{
+		// add a CAP_SYS_NICE capability to allow setting cpu affinity
+		"SYS_NICE",
+		// add CAP_SYS_ADMIN capability to allow virtiofs
+		"SYS_ADMIN",
```

Comment:
Do we want to only include `SYS_ADMIN` or all of them https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-controller/services/template.go#L1657?

Would it make sense to add these capabilities when the featureGate is requested?

Reply:
> Did you have a chance to see why openshift periodic on main was passing and we didn't see failure on this?

No. I am also not yet sure if it fixes the issue. Still waiting for @akrejcir to confirm that this is it.

Reply:
> Do we want to only include `SYS_ADMIN` or all of them https://github.com/kubevirt/kubevirt/blob/main/pkg/virt-controller/services/template.go#L1657?
> 
> Would it make sense to add these capabilities when the featureGate is requested?

That may make sense. Reduced the scope of the PR for now to only NET_BIND_SERVICE.

Reply:
> No. I am also not yet sure if it fixes the issue. Still waiting for @akrejcir to confirm that this is it.

Adding NET_BIND_SERVICE fixes the issue.